### PR TITLE
Add Gatsby

### PR DESCRIPTION
**Pull request by LekoArts**
We recently opened up our own Discord server to the public 👍 
https://github.com/gatsbyjs/gatsby

### CODE REVIEW

File: communities.json [hunk 1]
```diff
@@ -613,5 +613,13 @@
       "inviteCode": "Pnt5CuY",
       "githubUrl": "https://github.com/lutris"
     }
+    ,
```
Comment (SEAPUNK) [inline communities.json]: the comma needs to come directly after the `}` to conform to style rules

**Comment by LekoArts** [comment]
Fixed! Dunno why that happened 😅

**Comment by LekoArts** [comment]
@SEAPUNK 	
@SolarionFX 	
Please review 👍

**Comment by SolarionFX** [comment]
Thanks @LekoArts

**Pull request merged**